packages/metronome/build

```diff
@@ -21,7 +21,9 @@ PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/etc/check_time.env
 EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/metronome
+EnvironmentFile=-/opt/mesosphere/etc/metronome-extras
```

Comment:
What is in `metronome-extra`?

Reply:
nothing, this file does not exist in OSS. It's filled only in EE. We use this approach in other packages as well 
```ini
# git grep "EnvironmentFile=.*-extras"
packages/exhibitor/extra/dcos-exhibitor.service:EnvironmentFile=-/opt/mesosphere/etc/exhibitor-extras
packages/marathon/build:EnvironmentFile=-/opt/mesosphere/etc/marathon-extras
packages/mesos-dns/extra/dcos-mesos-dns.service:EnvironmentFile=-/opt/mesosphere/etc/mesos-dns-extras.env
packages/mesos/extra/dcos-mesos-master.service:EnvironmentFile=-/opt/mesosphere/etc/mesos-master-extras
packages/mesos/extra/dcos-mesos-slave-public.service:EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras
packages/mesos/extra/dcos-mesos-slave.service:EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras

```